examples/platform/silabs/efr32/display/lcd.cpp

```diff
@@ -180,8 +200,14 @@ void SilabsLCD::WriteQRCode()
             }
         }
     }
+#if (defined(EFR32MG24) && defined(WF200_WIFI))
+    pre_lcd_spi_transfer();
+#endif
 
     DMD_updateDisplay();
```

Comment:
Why not patch directly DMD_updateDisplay() to handle those preprocessor macros

Reply:
DMD_updateDisplay() is part of gecko SDK and we cant modify it.